conversiontracerbullet/src/main/java/org/stanford/AuthDBProperties.java

```diff
@@ -19,22 +16,22 @@
 
     private static Logger log = LogManager.getLogger(AuthDBProperties.class.getName());
 
-    private static final String DEFAULT_CONFIG_FILE = "/server.conf";
+    private static final String PROPERTY_RESOURCE = "/server.conf";
 
     private String server = null;
     private String service = null;
     private String userName = null;
     private String userPass = null;
-    private String propertyFile = null;
+    private Properties properties = null;
```

Comment:
No.  It is initialized by two different methods and those methods are specific to the type of constructor called.  Thereafter, it's available to the init method.  This is optimal already.